README.mkd

```diff
@@ -142,11 +142,9 @@ Both of those take a preceding count.
 
 To set your own mappings for these, for example `]h` and `[h`:
 
-In the lines below I have deleted brackets around GitGutter{Next,Prev}Hunk:
```

Comment:
Werid comment?!

Reply:
As @localstatic guessed, I accidentally committed the edits I made to just to have some changes to show in a screenshot.